feed.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>UrSpace | Home</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+    <link rel="stylesheet" href="./dist/css/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+    <script src="https://kit.fontawesome.com/c609dd8d17.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <header class="bg-primary">
+        <div class="container d-flex justify-content-between align-items-center p-3">
+            <div>
+                <a href="index.html">
+                    <img src="images/UrSpace Logo w.svg" alt="UrSpace Logo">
+                </a>
+            </div>
+            <div class="user-info"></div>
+        </div>
+    </header>
+
+    <main class="container">
+        <div class="row page-columns gap-3">
+
+            <div class="col py-4">
+                <div class="feed-left-column position-sticky">
+                    <nav class="nav"></nav>
+
+                    <!-- <div class="sort mt-5">
```

Comment:
If you aren't going to use it, put it in another branch and remove it from submission